### PR TITLE
[#128218253] passing context to MediaPlayer

### PR DESCRIPTION
**Pull request by ruslandinov**
This PR is required for successful completion of https://www.pivotaltracker.com/story/show/128218253

dash.js allows usage of custom context object in MediaPlayer.
This PR enables this feature in the bundle. The feature is used in https://github.com/streamroot/videojs5-dashjs-p2p-source-handler

### CODE REVIEW

File: lib/DashjsBundle.js [hunk 1]
```diff
@@ -4,20 +4,20 @@ require('../dashjs/all');
 
 import DashjsWrapper from './DashjsWrapper';
 
-function MediaPlayer(p2pConfig) {
+function MediaPlayer(context = {}, p2pConfig) {
```
Comment (maxbok) [inline lib/DashjsBundle.js]: Is it relevant to set a default value to `context` here?
To use it, we would need to call `MediaPlayer` without any arg, `p2pConfig` would be `undefined` and therefore no `streamrootKey`!

**Comment by maxbok** [comment]
Besides my comment, looks good to me!

**Comment by jlouazel** [comment]
👍 It's all good

File: lib/DashjsBundle.js (hunk 1 above)
Comment (ruslandinov) [inline lib/DashjsBundle.js]: You are right, better to swap these parameters.

Comment (ruslandinov) [inline lib/DashjsBundle.js]: You are right, absolutely useless to set it here. 
Doing it in this place https://github.com/streamroot/dashjs-p2p-wrapper/pull/27/files#diff-25d87cca845262366917c96a05ffeef6R17 is enough.

**Pull request merged**